src/Stratis.Bitcoin.Features.Consensus/Rules/ProvenHeaderRules/ProvenHeaderSizeRule.cs

```diff
@@ -10,8 +10,8 @@ namespace Stratis.Bitcoin.Features.Consensus.Rules.ProvenHeaderRules
     /// Rule to check if the serialized sizes of the proven header components, such as merkle proof (max 512 bytes),
     /// signature (max 80 bytes) and coinstake (max 1,000,000 bytes), do not exceed maximum possible size allocation.
     /// </summary>
-    /// <seealso cref="ProvenHeaderValidationConsensusRuleBase" />
-    public class ProvenHeaderSizeRule : ProvenHeaderValidationConsensusRuleBase
+    /// <seealso cref="ProvenHeaderRuleBase" />
+    public class ProvenHeaderSizeRule : ProvenHeaderRuleBase
```

Comment:
@dangershony But the whole purpose of having that base rule was to contain that `IsProvenHeaderActivated` helper property so that all rules related to PH can use it, otherwise I have to duplicate this code into every rule

Reply:
@dangershony Updated as per our discussion